### DISCUSSION TITLE
Error msg

### DISCUSSION THREAD

**Discussion by jo99jo** [soxoj/maigret#695] [Q&A] [answered]
I downloaded the Windows standalone exe.  Unzipped it and renamed it to maigret.

Then run from the install folder based on the docs:
**maigret.exe username -HP --tags en**

Keep getting error msg:
**[-] No sites to check, exiting!**

Tried adding options -a or --top-sites 500 but these options did not help.

The program is 40MB, so I am assuming that the sites are in the program.  

Any help as to what my error is?

**Comment by soxoj** [comment]
Very strange. Please, send result of command 'maigret.exe --version'.
@fen0s could you help to investigate it?

**Reply by jo99jo** [reply]
I didn't have the exe running from my C: drive, so moved the folder to C: root and then tried again.  Same result.

C:\Maigret>maigret_standalone.exe username -HP --tags en
[-] No sites to check, exiting!

C:\Maigret>maigret_standalone.exe --version
maigret_standalone.exe 0.4.4
Socid-extractor:  0.0.23
Aiohttp:  3.8.1
Requests:  2.28.1
Python:  3.7.5

**Reply by jo99jo** [reply]
I sent the version info you requested.  Does that help figure out any next steps?

**Reply by soxoj** [reply]
All is okay with your environment, but I've understand the problem.
Maigret doesn't have **language** tags, only **country** tags. So you should use `--tags us` instead of `--tags en`, and I should fix the documentation.
Thank you!

**Reply by jo99jo** [reply]
OK, that fixed the issue.  Thanks!

Yes, doc should be updated.

**Reply by soxoj** [reply, marked as answer]
Fixed in #715
